R/collectors.R

```diff
@@ -216,6 +217,9 @@ guess_parser <- function(x, locale = default_locale()) {
 #'
 #' # parse_factor generates a warning & problems
 #' x2 <- parse_factor(x, levels)
+#'
+#' # Using an argument of `NULL` will generate levels based on values of `x`
+#' x2 <- parse_factor(x, levels = NULL)
```

Comment:
According to SO we can depend on it being alphabetically http://stackoverflow.com/questions/7648756/is-the-order-of-iterating-through-stdmap-known-and-guaranteed-by-the-standard